core/proto/core.proto

```diff
@@ -0,0 +1,36 @@
+syntax = "proto2";
+
+package core;
+
+option go_package = "proto";
+
+message Challenge {
+	optional int64 id = 1;
+	optional string type = 2;
+	optional string status = 6;
+	// We use a string instead of an int64 millis / int32 nanos / timezone
+	// because having to decompose into 3 fields is extremely awkward.
+	optional string validatedTime = 8; // time.RFC3339Nano
```

Comment:
Commented offline: This is in the current challenge object but unused, so rather than propagate it here, let's do a separate PR removing it from `Challenge` (but keeping it in `challModel` for now so we don't have to do a migration).